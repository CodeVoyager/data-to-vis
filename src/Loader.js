import React from 'react'

export const Loader = (props) => (
    <div className="loader-wrapper">
        <div className="loader"></div>
    </div>
)

export default Loader