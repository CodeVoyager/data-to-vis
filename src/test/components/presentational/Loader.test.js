import React from 'react';
import ReactDOM from 'react-dom';
import Loader from '../../../components/presentational/Loader';
import renderer from 'react-test-renderer';

describe('Loader component', function () {
    it('renders without crashing', function () {
        const div = document.createElement('div');
        ReactDOM.render(<Loader/>, div);
    });

    it('renders correctly', function () {
        const tree1 = renderer.create(<Loader />).toJSON();
        expect(tree1).toMatchSnapshot();
    });
});