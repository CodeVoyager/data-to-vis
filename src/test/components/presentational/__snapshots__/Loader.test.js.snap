// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders correctly 1`] = `
<div
  className="loader-wrapper"
>
  <div
    className="loader"
  />
</div>
`;
